draft-ietf-tls-esni.md

```diff
@@ -793,13 +819,13 @@ up-to-date keys. Servers MAY supply multiple ECHConfig values of different
 versions. This allows a server to support multiple versions at once.
 
 Note that decryption failure could indicate a GREASE ECH extension (see
-{{grease-extensions}}), so it is necessary for servers to proceed with the
-connection and rely on the client to abort if ECH was required. In particular,
-the unrecognized value alone does not indicate a misconfigured ECH
-advertisement ({{misconfiguration}}). Instead, servers can measure occurrences
-of the "ech_required" alert to detect this case.
+{{grease-ech}}), so it is necessary for servers to proceed with the connection
+and rely on the client to abort if ECH was required. In particular, the
+unrecognized value alone does not indicate a misconfigured ECH advertisement
+({{misconfiguration}}). Instead, servers can measure occurrences of the
+"ech_required" alert to detect this case.
 
-### HelloRetryRequest {#server-hrr}
+### Handling Hello Retry Request {#server-hrr}
 
 In case a HelloRetryRequest (HRR) is sent, the client-facing server MUST
```

Comment:
Point 1 in this section also doesn't really account for empty vs non-empty extensions. (Which is why I was pondering if we should just use a separate codepoint. Although #352 simplifies this anyway, so we could also leave it alone and fix it separately. But in that case maybe we should leave #348 open. This PR is currently set to close it.)

Reply:
Let's leave the issue open. (I changed the top comment above to reflect this.)

Reply:
SGTM.

Reply:
(Oh, apparently I can't mark threads as resolved. Feel free to close this out.)